src/coreclr/src/tools/crossgen2/ILCompiler.Reflection.ReadyToRun/ReadyToRunSignature.cs

```diff
@@ -612,41 +1026,41 @@ private void EmitInlineSignatureBinaryBytes(StringBuilder builder, int count)
                     {
                         builder.Append('-');
                     }
-                    builder.Append(_image[_offset - count + index].ToString("x2"));
+                    builder.Append(_image[Offset - count + index].ToString("x2"));
                 }
                 builder.Append("-");
             }
         }
 
         private uint ReadUIntAndEmitInlineSignatureBinary(StringBuilder builder)
         {
-            int startOffset = _offset;
+            int startOffset = Offset;
             uint value = ReadUInt();
             EmitInlineSignatureBinaryForm(builder, startOffset);
             return value;
         }
 
         private int ReadIntAndEmitInlineSignatureBinary(StringBuilder builder)
         {
-            int startOffset = _offset;
+            int startOffset = Offset;
             int value = ReadInt();
             EmitInlineSignatureBinaryForm(builder, startOffset);
             return value;
         }
 
         private uint ReadTokenAndEmitInlineSignatureBinary(StringBuilder builder)
         {
-            int startOffset = _offset;
+            int startOffset = Offset;
             uint value = ReadToken();
             EmitInlineSignatureBinaryForm(builder, startOffset);
             return value;
         }
 
         private void EmitSignatureBinaryFrom(StringBuilder builder, int startOffset)
```

Comment:
Should this be `EmitSignatureBinaryForm`?

Reply:
Eh, I actually think the From naming is more accurate. It specifies to dump a signature from and offset to the current offset. In any case, I haven't changed those names in this PR, and given that the change is already too large, I don't want to make unnecessary changes.